sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_common/_constants.py

```diff
@@ -27,4 +27,9 @@
 
 
 class SchemaFormat(str, Enum):
+    """
+    Represents the format of the schema to be stored by the Schema Registry service.
+    """
+
     AVRO = "avro"
+    """Represents the Apache Avro schema format."""
```

Comment:
tbh, I just copied the tables class docstring here: https://github.com/Azure/azure-sdk-for-python/blob/c8291ac6cb0dbd865da03a88dd2bcb9279e2c4a6/sdk/tables/azure-data-tables/azure/data/tables/_entity.py#L27

I assumed that it would be rendered, but
the docs that were generated by the CI didn't contain this line. Although I'm not sure if that's just b/c they didn't generate properly since the CI failed.

I can remove it if you prefer?

Reply:
@yunhaoling  just checked, and this line does render!